### PR TITLE
Added notebook and updated manifest

### PR DESCRIPTION
**Pull request by Niels1006**

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -20,7 +21,7 @@ StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
 
 [compat]
+GraphMakie = "0.5.5"
 MINDFul = "0.2"
 NestedGraphMakie = "0.1"
-GraphMakie = "0.5.5"
 julia = "1.9"
```
Comment (filchristou) [inline Project.toml]: Can you add julia 1.10 here ?
```suggestion
julia = "1.9, 1.10"
```

File: Manifest.toml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.9.2"
+julia_version = "1.9.4"
```
Comment (filchristou) [inline Manifest.toml]: Can you try that with julia 1.10 ?
```suggestion
julia_version = "1.10"
```

**Review by filchristou — CHANGES_REQUESTED** [review]
Did you think of a different way to add the tsParticles file other than putting it in the `./data/` directory ?

**Comment by Niels1006** [comment]
> Did you think of a different way to add the tsParticles file other than putting it in the `./data/` directory ?

No, but I think that will do it for now.

**Review by filchristou — APPROVED** [review]

**Pull request merged**